environments/demo/backend_lb_config.yaml

```diff
@@ -396,6 +396,8 @@ gateways:
         component: case-api
       - product: sptribs
```

Comment:
Can we remove the dss-backend? Or, if not yet, can we raise a followup ticket to do so when we can?

Reply:
we will do that by separate PR as we agreed